### PR TITLE
fix: fields & combined mode in vue3

### PR DESCRIPTION
**Pull request by cb-farhanh**
#### Description
- Fix fields & combined mode to work in Vue3.

### CODE REVIEW

File: chargebee-js-vue/src/components/Element.vue [hunk 1]
```diff
@@ -47,13 +47,14 @@ export default {
...
         this.field = cbComponent.createField(this.id, options).at(`#${this.elementId}`)
-        if (this.$parent.onMount) this.$parent.onMount()
+        if (this.$parent.onMount) this.$parent.onMount();
 
         // Attach listeners if any
         this.attachListener('ready')
         this.attachListener('focus')
         this.attachListener('blur')
         this.attachListener('change')
+        this.initialized = true;
```
Comment (cb-dinesh) [inline chargebee-js-vue/src/components/Element.vue:57]: `initialized` variable to be added to the `data` attributes

File: chargebee-js-vue/src/components/Element.vue [hunk 2]
```diff
@@ -47,13 +47,14 @@ export default {
       if (cbComponent) {
         const options = this.fieldOptions;
         this.field = cbComponent.createField(this.id, options).at(`#${this.elementId}`)
-        if (this.$parent.onMount) this.$parent.onMount()
+        if (this.$parent.onMount) this.$parent.onMount();
```
Comment (cb-dinesh) [inline chargebee-js-vue/src/components/Element.vue:50]: Ideally, This should be an emit event, and as a response we should be calling onMount method from the parent.

File: chargebee-js-vue/src/components/Element.vue (hunk 1 above)
Comment (cb-farhanh) [reply chargebee-js-vue/src/components/Element.vue:58]: Done.

File: chargebee-js-vue/src/components/Element.vue (hunk 2 above)
Comment (cb-farhanh) [reply chargebee-js-vue/src/components/Element.vue:51]: Noted.
https://mychargebee.atlassian.net/browse/CHKOUTENGG-42575

**Review by cb-dinesh — APPROVED** [review]

**Pull request merged**